kong/plugins/apifortress/fortress_serializer.lua

```diff
@@ -0,0 +1,99 @@
+-- ©2016 API Fortress Inc.
+
+local _M = {}
+function string.starts(String,Start)
```

Comment:
here you are extending the standard string library, which would be considered bad practice. Instead, use `stringy` which is one of Kong's dependencies and provide such facilities (starts/ends).